cryptosuite/bbsplussignatureproofsuite.go

```diff
@@ -0,0 +1,248 @@
+package cryptosuite
```

Comment:
this is where selective disclosure comes in; not yet implemented

Reply:
Still need to go over this file, but wanted to get some early feedback out.